plugins/discourse-post-voting/assets/javascripts/discourse/components/reviewable-refresh/post-voting-comment.gjs

```diff
@@ -0,0 +1,35 @@
+import ReviewablePostEdits from "discourse/components/reviewable-post-edits";
+import ReviewableCreatedBy from "discourse/components/reviewable-refresh/created-by";
+import ReviewableTopicLink from "discourse/components/reviewable-refresh/topic-link";
+import highlightWatchedWords from "discourse/lib/highlight-watched-words";
+import { i18n } from "discourse-i18n";
+
+export default <template>
+  <div class="review-item__meta-content">
```

Comment:
My hunch is we might want all this markup and class names more abstracted in a core component so that we don't break plugins ~if~ when we update core. But that might be better reserved for a follow-up.